gramineproject/gramine
# [tests] Fix installation of Musl tests

## Description of the changes 

The previous script moved the test binaries out of the build directory, instead of copying them. As a result, every "rebuild and install" cycle ended up rebuilding all the Musl binaries, even if their dependencies didn't change.


## How to test this PR? 

Rebuild and install Gramine twice. The second rebuild should only rebuild about 4 targets (because of the `vcs_tag`) instead of rebuilding 99 of them.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/300)
